lua/nvim-treesitter/install.lua

```diff
@@ -50,6 +50,10 @@ local function get_installed_revision(lang)
   end
 end
 
+local function is_installed(lang)
+  return #api.nvim_get_runtime_file('parser/'..lang..'.so', false) > 0
+end
```

Comment:
Isn't this the same as `has_parser` or do you explicitly want to circumvent any caching with own local implementation?